website/docs/cli/commands/test.mdx

```diff
@@ -1,12 +1,284 @@
 ---
-description: Part of the ongoing design research for module integration testing.
+description: >-
+    The tofu test command performs integration tests of OpenTofu modules.
 ---
 
 # Command: test
 
-The `tofu test` command is currently serving as part of
-[the module integration testing experiment](/docs/language/modules/testing-experiment).
+The `tofu test` command runs integration tests to ensure intended behavior of your OpenTofu configuration. It creates real
+infrastructure, asserts conditional checks against the attributes of provisioned resources and attempts
+to clean up the testing infrastructure upon completion.
 
-It's not ready for routine use, but if you'd be interested in trying the
-prototype functionality then we'd love to hear your feedback. See the
-experiment details page linked above for more information.
+## Usage
+
+Usage: `tofu test [options]`
+
+The `test` command executes the automated integration tests defined in `.tftest.hcl` files which OpenTofu will read
+from the current configuration and testing directories.
+
+OpenTofu will create real infrastructure according to the definition in the tested configuration. Then, it will
+sequentially run the test blocks to perform conditional checks and assertions against attributes of provisioned resources.
+Upon tests completion, OpenTofu will attempt to clean up created resources.
+
+_Note_: monitor the test output for successful cleanup.
```

Comment:
Should this maybe be an info box instead of a Note?

Reply:
Thanks for suggestion Kuba! Implemented.